### ISSUE TITLE
Update predicted distribution (and delta) colors

### ISSUE THREAD

**Issue by jcoliver** [Big-Biodiversity-Collaborative/SwallowtailClimateChange#105]
The current green, light blue, purple color scheme for predicted distribution maps may not be idea. Perhaps the [Dark 2 scheme](https://colorbrewer2.org/#type=qualitative&scheme=Dark2&n=3) would work for the distributions.

![image](https://github.com/user-attachments/assets/13948afd-f03c-4405-a390-ccf641e771dd)
 
And the purple orange [PuOr](https://colorbrewer2.org/#type=diverging&scheme=PuOr&n=3) scheme for the deltas.

![image](https://github.com/user-attachments/assets/70082a19-df78-44c0-ae1a-10059ee2b3b0)

Or maybe brown, blue, green [BrBG](https://colorbrewer2.org/#type=diverging&scheme=BrBG&n=3)?

![image](https://github.com/user-attachments/assets/c3a87e2d-081b-42a7-bd88-5a6ffacd6309)

**Comment by ezylstra** [comment]
I'll work on this Monday. Note that if we change colors in distribution/overlap maps, we'll also need to change colors in workflow.

**Comment by ezylstra** [comment]
Tried the dark color palette for the overlap maps, but I think it made it more difficult to read the maps. Tried some other options but think it might be better to go with the Paired palette from color brewer that we've used in presentations (which is still colorblind friendly). 

As for the delta maps, I liked the suggestions above, but they don't work with a light gray middle given that we have a light gray for "absent".  Best option I found is a tweaked version of the RdYlBu palette (that has a yellow middle, that would be used to represent stable areas).
